services/logs/filter_test.go

```diff
@@ -168,33 +168,71 @@ func TestIDFilter(t *testing.T) {
 		desc: "wildcard all logs",
 		id:   mustHash(blocks[0]),
 		criteria: FilterCriteria{
-			Addresses: []common.Address{lgs[0].Address},
```

Comment:
why is filter by address removed? we need to have one test at least that filters by address

Reply:
We already have such a test below, see: https://github.com/onflow/flow-evm-gateway/pull/423/files#diff-3dcf94bdc0a09d411408b6d264b30c0f2ad933cdf8dcf518d30a6718640a97a6R222-R227 .
It's just that the `wildcard all logs` filter was basically a duplicate test, as it provided an address.
So it wasn't really a wildcard filter criteria.